src/Home.js

```diff
@@ -1,15 +1,72 @@
 import React, { Component } from 'react';
- 
+import Container from 'react-bootstrap/Container'
+import Row from 'react-bootstrap/Row'
+import Col from 'react-bootstrap/Col'
+import logo from './Images/ScrewtheReviewLogo.png'
+import searchIcon from './Images/SearchIcon.png'
+import wheelIcon from './Images/WheelIcon.png'
+import contactIcon from './Images/ContactIcon.png'
+import aboutIcon from './Images/AboutIcon.png'
+
 class Home extends Component {
     render() {
         return (
-            <div>
-                <h1>Home</h1>
-                <p>Home page body content</p>
-                <p>CI/CD Deployment Test</p>
-            </div>
+            <Container fluid id="home-container">
+                <Row className="align-items-center">
+                    {/* Column 1: Logo, Auth0 Widgets and Description*/}
+                    <Col id="landing_col_1" align='center' style={{paddingBottom: 20}} lg={6} md={12} sm={12} xs={12}>
+                        <img  src={logo} alt="Logo" id ="logo"/>
+                        <h5 id="welcome"> Welcome to Screw the Review!</h5>
+                        <p id="info"> Screw the Review provides new experiences by randomly generating activities. The Roulette
```

Comment:
Can we really say we "randomly generate activities?"

Also, Roulette still takes location into account